uwpx.doap

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="https://uwpx.org/files/doap-style.xsl" type="text/xsl"?>
+<rdf:RDF xmlns:rdf='http://www.w3.org/1999/02/22-rdf-syntax-ns#' xmlns='http://usefulinc.com/ns/doap#' xmlns:xmpp='https://linkmauve.fr/ns/xmpp-doap#' xmlns:schema='https://schema.org' xmlns:foaf="http://xmlns.com/foaf/0.1/">
+  <Project>
+
+    <name>UWPX</name>
+    <short-name>UWPX</short-name>
+
+    <created>2018-01-01</created>
+
+    <shortdesc xml:lang="en">UWPX is a secure and open-source XMPP app for all your UWP (Windows 10) devices </shortdesc>
+    <description xml:lang="en">
+    UWPX is a app for your UWP (Windows 10) devices. It implements the Extensible Messaging and Presence Protocol (XMPP). At the moment UWPX is still in ALPHA state so expect regular crashes and unexpected behavior. If you encounter bugs or even crashes and somehow can reproduce them please report them over at GitHub. UWPX is completely Open Source. It's licensed under Mozilla Public License 2.0. Feel free to clone, change and submit features and bug fixes - there are always bugs waiting to be fixed. You are not limited to normal chats with only one recipient. UWPX supports XEP-0045: Multi-User Chat (MUC). Just click "Add Group Chat (MUC)", enter the room name and voila you created a new or joind an existing MUC. 
+    </description>
+
+    <homepage rdf:resource="https://uwpx.org/" />
+    <bug-database rdf:resource="https://github.com/UWPX/UWPX-Client/issues" />
+    <license rdf:resource="https://github.com/UWPX/UWPX-Client/blob/master/LICENSE" />
+    <schema:documentation rdf:resource="https://github.com/UWPX/UWPX-Client" />
+
+    <!-- See https://github.com/ewilderj/doap/issues/49 -->
+    <language>en</language>
+
+    <programming-language>C#</programming-language>
+    <os>Windows 10</os>
+    <os>UWP</os>
```

Comment:
I know this is technically correct, but maybe adding `<os>Windows</os>` would minimize confusion.

Reply:
Ah, strike that, I didn't see `Windows 10`.

Reply:
Replaced it with `Windows`, since Windows 11 will be a thing 🙃.